platform/podjs/src/browserPod.ts

```diff
@@ -43,6 +56,11 @@ class LocalStoragePolyIn implements PolyIn {
         });
     }
 
+    /**
+     * It returns a list of quads that match the given matcher
+     * @param matcher - Partial<Matcher>
+     * @returns An array of quads.
```

Comment:
Possibly be a more precise here: `RDF.Quad`. It's probably obvious since this is async, but maybe it's best to precise it's a Promise what's returned.